scripts/ui-injector.js

```diff
@@ -21,6 +21,15 @@ waitForElement(`#${TIME_RECORDING_MAIN_SECTION_ID} > div:nth-of-type(2)`)
 })
 .catch(error => console.info('Error while finding the Time Recording container: ', error.message));
 
+// Inject duration shortcuts before before the original duration data-element
+waitForElement(`#${TIME_RECORDING_DURATION_DATA_ID}`)
```

Comment:
After rebasing with current main, this can be the new selector
```suggestion
waitForElement(`${PROJECT_SECTION_SELECTOR} > div:nth-of-type(8)`)
```